includes/definitions/discovery/fujifilmprinter.yaml

```diff
@@ -0,0 +1,5 @@
+modules:
+    os:
+        sysDescr_regex: '/FUJIFILM (?<hardware>[^;]+)/'
+        hardware: .1.3.6.1.4.1.297.1.11.93.2.1
```

Comment:
this line seems wrong?

Reply:
Sorry, I had a misunderstanding about the yaml, I've fixed it.